pilot/pkg/kube/inject/testdata/auth.cert-dir.yaml.injected

```diff
@@ -83,6 +83,10 @@ spec:
         - "15001"
         - -u
         - "1337"
+        - -i
+        - '*'
```

Comment:
Confused - why isn't it "*" (it seems the template is using double quotes)

Reply:
This is an artifact of the template processing.  When I use double quotes around it in the template, it results in a single quote.  I'd be happy to change it if you know what I need to change in the template.